lib/rubocop/cop/style/redundant_regexp_quantifiers.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for redundant quantifiers inside Regexp literals.
+      #
+      # @example
+      #   # bad
+      #   /(?:x+)+/
```

Comment:
unfortunately, some of these examples emit warnings during the `documentation_syntax_check` task.

is there an established way to prevent this?

Reply:
Don't think so. @koic can correct me about this, but in general we have a ton of AsciiDocs warnings these days that we'll have to address at some point.